pycaret/time_series/forecasting/oop.py

```diff
@@ -1075,7 +1075,9 @@ def setup(
         numeric_imputation_target: Optional[Union[int, float, str]] = None,
         numeric_imputation_exogenous: Optional[Union[int, float, str]] = None,
         transform_target: Optional[str] = None,
+        transform_target_kwargs: Optional[dict] = None,
```

Comment:
I will have to check with the rest of the team if we want to add kwargs or dedicated arguments.

Things to consider

1. Consistency with the previous interface for regression/classification.
2. We also plan to add a custom pipeline option as in regression/classification so the kwargs may be a moot point if we provide some dedicated arguments (i.e. users can specify their custom pipeline if we want to customize beyond what we provide).

I will try to get you an answer by the weekend.

Reply:
@AJarman : Interface feedback left here: https://github.com/pycaret/pycaret/issues/1766#issuecomment-1094078302